wsi/physicaldisplay.cpp

```diff
@@ -110,21 +110,16 @@ void PhysicalDisplay::DisConnect() {
 
   display_state_ &= ~kDisconnectionInProgress;
 
-  if (!(display_state_ & kConnected)) {
+  if (!display_queue_->Exit(power_mode_)) {
     SPIN_UNLOCK(modeset_lock_);
-
     return;
   }
+
   IHOTPLUGEVENTTRACE(
       "PhysicalDisplay DisConnect called for Display: %p hotplugdisplayid: %d "
       "\n",
       this, hot_plug_display_id_);
   display_state_ |= kNotifyClient;
-
-  if (power_mode_ != kOff) {
-    display_queue_->SetPowerMode(kOff);
```

Comment:
Here we can call Exit API. It doesn't need to take in powermode, it would just mean display has been physically disconnected

Reply:
Ok I will add this exit here and rename into HandleDisconnect()